docs/release/supported_platforms.md

```diff
@@ -11,13 +11,10 @@ for 64-bit Intel architectures according to the following table:
 
 | Platform                   | OSG 3.5  | OSG 3.4  |
```

Comment:
3.4's out of support -- should we change this to 3.5 and 3.6?

Reply:
I'll add 3.6 in another PR